src/state/learnocaml_data.mli

```diff
@@ -206,6 +206,13 @@ module Exercise: sig
     val get_status:
       Token.t -> assignments -> status
 
+    val is_open_assg_globally:
+      assignments -> [> `Open | `OpenAssg | `Assigned | `Closed]
+
+    val set_close_assg_globally: assignments -> assignments
+
+    val set_open_assg_globally: assignments -> assignments
```

Comment:
I feel the need for a docstring here, as the semantics of "OpenAssg" is unclear to me.

Reply:
Good point.